BPS0026_MVC_WEB/BPS0026_MVC_WEB/Migrations/202109230014448_InitialCreate.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace BPS0026_MVC_WEB.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class InitialCreate : DbMigration
+    {
+        public override void Up()
+        {
+            CreateTable(
+                "dbo.Customers",
+                c => new
+                    {
+                        CustomerId = c.Int(nullable: false, identity: true),
+                        No = c.String(),
+                        Date = c.String(),
+                        CustomerName = c.String(),
+                        Amount = c.Int(nullable: false),
+                    })
+                .PrimaryKey(t => t.CustomerId);
+            
+        }
```

Comment:
Please use Fluent API In Code First Approach.

Reply:
Done.
Salamat sa advice :)